modules/scale-operator-and-deploymentconfigs.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-using-oadp.adoc assembly
+
+[id="scale-operator-and-deploymentconfigs_{context}"]
+= Scale operator and deploymentConfigs
+
+Scale up the 3scale operator and any deploymentConfigs that were manually scaled down. After a few minutes, 3scale installation should be fully functional, and its state should match the backed-up state.
+
+.Prerequisites
+
+* Ensure that there are no DeploymentConfigs (DCs) scaled up already, or any extra pods are running. 
+There might be some system-mysql or backend-redis pods running detached from DC after restoration, which can be removed after the restoration is successful.
+
+
+.Procedure
+
+. Scale operator and DC:
++
+[source,terminal]
+----
+$ oc scale deployment threescale-operator-controller-manager-v2 --replicas=1 -n threescale
+----
++
```

Comment:
. Scale the deployment configuration (DC) with a script by running the following commands: